src/oracles/CoinbaseOracle.sol

```diff
@@ -25,31 +29,30 @@ contract CoinbaseOracle is IAggregatorV3SourceCoinbase {
     /**
      * @notice Constructor to initialize the CoinbaseOracle contract.
      * @param _decimals The number of decimals in the reported price.
+     * @param _dataKind The kind of data that the oracle will report, expected to be "prices" but open for changes.
      * @param _reporter The address of the reporter allowed to push price data.
      */
-    constructor(uint8 _decimals, address _reporter) {
+    constructor(uint8 _decimals, string memory _dataKind, address _reporter) {
```

Comment:
We now can pass a data kind in the constructor

Reply:
if it's always `prices` then we should not yet the deployer to define this, I think? if it should always be a hard-coded value then we should hard-code it.